config/config.exs

```diff
@@ -33,7 +33,8 @@ config :realtime_signs,
   aws_client: ExAws,
   s3_client: ExAws.S3,
   number_of_http_updaters: 4,
-  restart_fn: &Engine.Health.restart_noop/0
+  restart_fn: &Engine.Health.restart_noop/0,
+  monitoring_api_key: "local_monitoring_api_key"
```

Comment:
Since this is security-related, I don't think we should provide a default value. It we set the config unconditionally from the environment, then it will be `nil` if the environment variable is missing, which should not be possible to get past (since header values are always strings). So the API would be inaccessible, which seems like a good default behavior.

Reply:
Hmm yeah that's a good point! Will remove this default